libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookHelper.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Adapters.Facebook
+{
+    public static class FacebookHelper
+    {
+        /// <summary>
+        /// Converts an Activity object to a Facebook messenger outbound message ready for the API.
+        /// </summary>
+        /// <param name="activity">The activity to be converted to Facebook message.</param>
+        /// <returns>The resulting message.</returns>
+        public static FacebookMessage ActivityToFacebook(Activity activity)
+        {
```

Comment:
Add validation for activity == null.

Reply:
not quite sure I agree with that but, ok

Reply:
It's what we did for previous implementations of similat methods.